### ISSUE TITLE
Local workspace call workspace avatar blob download request, it's not necessary

### ISSUE THREAD

**Issue by tzhangchi** [toeverything/AFFiNE#920]

**Comment by tzhangchi** [comment]
```
/api/workspace/140ee5ed-17d1-47e1-86e8-47fc690ccf5c/blob/ECvPm5_4ksjh7WsV8bA00n8MW_WwwFrLyotv4mTwMHk=
```



![Image](https://user-images.githubusercontent.com/5910926/217733106-6547ff37-6c11-4749-8e49-92fc9209ecbd.png)

**Comment by DiamondThree** [comment]
Closed by #925
